### DISCUSSION TITLE
Pancake lottery close is not working

### DISCUSSION THREAD

**Discussion by kryptoDev0777** [community/community#36435] [Other Feature Feedback, Questions, & Ideas] [answered]
Schedule project is not working on time.
I want to close and start every 6 hours

**Comment by 0xlightfury** [comment, marked as answer]
hi
have you sent link token to calculator?

**Reply by kryptoDev0777** [reply]
oh, I didn't, I will check again

**Comment by 0xlightfury** [comment]
yes, that can be issue

**Reply by kryptoDev0777** [reply]
yeah, you are right, it works now
